modules/mobile/docs/openapi.yaml

```diff
@@ -131,6 +131,68 @@ paths:
       security:
         - Bearer: []
       summary: /v0/appeal/{id}
+  /v0/appointment_requests/:appointment_request_id/messages:
```

Comment:
one more small cosmetic thing. Other endpoints use {} to denote a variable rather then colon

Reply:
Good eye. Thank you.